ui/src/testing/stubs/workspace-service-stub.ts

```diff
@@ -57,6 +60,7 @@ export class WorkspacesServiceStub {
     }
   ];
   notebookList: FileDetail[];
+  resourceList: RecentResource[];
```

Comment:
Yup L331

Reply:
I saw that you were setting it in this file - but I still don't see why it would be needed in WorkspaceServiceStub nor do I see it being read anywhere